src/cluster/cluster.go

```diff
@@ -175,9 +229,9 @@ func (c *Cluster) JoinNetwork(URL string) error {
 
 //generateUniqueID returns a unique id for asigning to a new microcontroller
 func (c *Cluster) generateUniqueID() int {
-	randID := rand.Intn(100)
+	randID := rand.Intn(255)
```

Comment:
Could we add a a future TODO here for making this an ENV var or something more easily configurable.

Reply:
This is done